packages/vaadin-combo-box/test/dropdown.test.js

```diff
@@ -1,5 +1,6 @@
 import { expect } from '@esm-bundle/chai';
 import { fixtureSync } from '@vaadin/testing-helpers';
+import '@vaadin/vaadin-template-renderer';
```

Comment:
This test suite does not cover `vaadin-combo-box` and only the dropdown, do we need this here as well?

Reply:
You are right, it isn't necessary. I added this import because of the further used `<template />` tag, but it also isn't necessary.

Reply:
Removed.